bundles/net.tourbook/src/net/tourbook/ui/TreeColumnFactory.java

```diff
@@ -646,6 +649,26 @@ public TreeColumnDefinition createColumn(final ColumnManager columnManager,
          }
       };
 
+
+      NUTRITION_NUM_PRODUCTS = new TreeColumnFactory() {
+         @Override
+         public TreeColumnDefinition createColumn(final ColumnManager columnManager,
+                                                   final PixelConverter pixelConverter) {
+
+            final TreeColumnDefinition colDef = new TreeColumnDefinition(columnManager, NUTRITION_NUM_PRODUCTS_ID, SWT.TRAIL);
+
+            colDef.setColumnCategory(           Messages.ColumnFactory_Category_Nutrition);
```

Comment:
The columns are sorted by category and name, the nutrition column is just within the device category columns

Reply:
Good catch. 
I moved it after the category `Motion`